backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/database/repos/AcceptingStoresRepository.kt

```diff
@@ -76,6 +84,57 @@ object AcceptingStoresRepository {
             .limit(limit, offset)
     }
 
+    fun determineRemovableAcceptingStoreId(acceptingStore: AcceptingStore, project: ProjectEntity): Int? {
```

Comment:
optional:
I would suggest naming it smthg like getIdIfExists(), because the current name says more about what the function is currently used for rather than what it actually does.